### ISSUE TITLE
Echo Invalid Certificate

### ISSUE THREAD

**Issue by jason00111** [LearnersGuild/los#93]
Invalid certificate error when I try to access echo.  See screenshots below.
I've tried logging out and back in, restarting computer, same error for a few days now.
On Mac, Chrome is up to date.

<img width="621" alt="screen shot 2016-10-11 at 8 42 09 pm" src="https://cloud.githubusercontent.com/assets/20765448/19297027/df539254-8ff5-11e6-9aeb-90013d9ba88e.png">
<img width="1178" alt="screen shot 2016-10-11 at 8 39 01 pm" src="https://cloud.githubusercontent.com/assets/20765448/19297038/f29908ee-8ff5-11e6-9cb1-97fbb39d49d1.png">
<img width="354" alt="screen shot 2016-10-11 at 8 41 22 pm" src="https://cloud.githubusercontent.com/assets/20765448/19297040/f619ba4a-8ff5-11e6-8d90-a25efbd8fc3a.png">
<img width="462" alt="screen shot 2016-10-11 at 8 44 15 pm" src="https://cloud.githubusercontent.com/assets/20765448/19297045/f90f477e-8ff5-11e6-98ff-4abd604cb639.png">

**Comment by jason00111** [comment]
Also see issue #75

**Comment by jeffreywescott** [comment]
Hi, Jason. As described in #75, our certs are definitely valid and up to date. This is most likely an issue related to a cached root cert, [as described here](https://apple.stackexchange.com/questions/134601/chrome-showing-the-sites-security-certificate-is-not-trusted-for-github).

Try opening Keychain Access, searching for "COMODO" and deleting any of those certificates, then restart chrome, flush all caches, and see if it still happens.

**Comment by jason00111** [comment]
I managed to delete the certificate with the name "COMODO" (I had to go into recovery mode and run `csrutil disable` before I could do this) but that didn't solve the problem.

I flushed all caches in Chrome and deleted the folder `~/Library/Application Support/Google/Chrome` but that didn't solve the problem.

Uninstalling and reinstalling Chrome **DID** solve the problem.  I was able to keep my bookmarks and extensions by backing up my `~/Library/Application Support/Google/Chrome` folder and restoring it after the reinstall.

Thanks for your help.  Hopefully the next person who encounters this issue can learn from this.
